_pages/about.html

```diff
@@ -37,19 +37,17 @@ <h4 class="teams__card-header">Scrum</h4>
 <section>
   <h2 class="section__title">International. Interdisciplinary. Connected.</h2>
 
-  <p class="section__body">More than 100 creative minds are spread over these locations:</p>
+  <p class="section__body">These are our four offices where we come together:</p>
 
   <div class="card">
 
     <div class="cities">
 
       <div class="city">
         <div class="js-city-slider">
-          <div class="city__header" style="background-image: url('/assets/img/pages/about/hamburg.jpg')"></div>
-          <div class="city__header" style="background-image: url('/assets/img/pages/about/hamburg-1.jpg')"></div>
-          <div class="city__header" style="background-image: url('/assets/img/pages/about/hamburg-2.jpg')"></div>
-          <div class="city__header" style="background-image: url('/assets/img/pages/about/hamburg-3.jpg')"></div>
-          <div class="city__header" style="background-image: url('/assets/img/pages/about/hamburg-4.jpg')"></div>
+          <div class="city__header" style="background-image: url('/assets/img/pages/about/hamburg-1-1.jpg')"></div>
+          <div class="city__header" style="background-image: url('/assets/img/pages/about/hamburg-2-1.jpg')"></div>
+          <div class="city__header" style="background-image: url('/assets/img/pages/about/hamburg-3-1.jpg')"></div>
```

Comment:
@AliceStatler Since we moved to one floor and the picture shows the social room at the HP I would suggest to use another picture if available.
Maybe we have a picture of our concept store? Or a picture of Planten un Blomen, since it is right next to the office.

Reply:
@dkluever22 As soon as the "new" Hamburg office is more presentable and the transition is done I also want to add more pictures. During spring/summer we could also add some pictures of Planten un Blomen.